reference/global-api-information.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: Global API Information
+---
+
+It is possible to provide some global API information in the `ApiDefinition.define` block with the `info` method. You may define this metadata for all versions of your API, or only for a specific version. This is purely informational output in the JSON files, it is *not* used in the doc browser or anywhere else yet.
+
+The following directives are supported:
+
+ * `name`
+ * `title`
+ * `description`
+ * `base_path`
+
+Below is a basic ApiDefinition that defines global info as well info for a specific version:
+
+{% highlight ruby %}
+Praxis::ApiDefinition.define
+  
+  info do
+    name 'Some App'
+    title 'An example Praxis application'
```

Comment:
it might be nice to have the global info define a description that is more loose, i.e., 
description "This is an App API"

...and then maybe place a comment line above the description for 1.0 below, saying that would overwrite the globally inherited descriptino...